src/js/libs/components/fetch.js

```diff
@@ -0,0 +1,49 @@
+function createNode(element) {
+    return document.createElement(element);
+}
+
+function append(parent, el) {
+  return parent.appendChild(el);
+}
+
+const tr1 = document.getElementById('loadBalancer1');
+
+export function getTableData() {
+    const response = fetch("https://westend.vedran.nodefactory.io/metrics")
```

Comment:
to make it easier for third party people to add their load balancer to the list, could you create in some file constant like:
```js
export const loadbalancers = [
{
   name: 'NodeFactory',
   network: 'Westend',
   url: 'wss://westend.vedran.nodefactory.io/ws'
   metricsUrl: 'https://westend.vedran.nodefactory.io/metrics'
}
]
```
and use that constant here to load data for all load balancers in this array?